### DISCUSSION TITLE
Show generated seed number for "Random" mod

### DISCUSSION THREAD

**Discussion by xDololow** [ppy/osu#15462] [Ideas]
The title says it. The field is empty now and you can't save the seed you liked or wanted to share.

**Comment by smoogipoo** [comment]
The seed is currently generated when play is started, and changes every time you retry. It should probably show in the tooltip if it doesn't already, and eventually I believe we'll have an area on the results screen dedicated to listing all mod settings.

**Reply by xDololow** [reply]
> It should probably show in the tooltip if it doesn't already

_its empty
![image](https://user-images.githubusercontent.com/26763912/140281070-eadf90a0-593e-42b5-b3d7-5cb842326bba.png)_

edit: I see what you meant by the tooltip. but its kinda not intuitive 
![image](https://user-images.githubusercontent.com/26763912/140281636-5d464971-0962-4b0f-a4a0-c33916e9731d.png)
